packages/client/src/widgets/AccountCredit/ui/CreditCategoryFilter.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Select } from '@gds/component'
+import { IconArrowDown } from '@gds/icon'
+
+interface CategoryItem {
+   label: string
+   id: string
+}
+
+interface Props {
+   selectedCategory: CategoryItem
+   onSelectCategory: ({ label, id }: { label: string; id: string }) => void
```

Comment:
P4

바로 위에 타입이 있어서 바꾸는 게 좋을 것 같습니다!

```suggestion
   onSelectCategory: ({ label, id }: CategoryItem) => void
```